app/assets/javascripts/miq_application.js

```diff
@@ -835,6 +835,9 @@ function miqAjaxAuth(url) {
 
     miqEnableLoginFields(true);
     miqSparkleOff();
+  })
+  .then(function() {
+    return API.ws_init;
```

Comment:
I guess you meant `return API.ws_init();` ? ;) But this is not the right place, since the `/dashboard/authenticate` request returns a redirect - so .. this call will get cancelled or not even started given the right phase of the moon ;).

It should happen (and finish) before doing that `miqJqueryRequest`, or be done from `API.init..`